external-scripts.json

```diff
@@ -1 +1 @@
-["hubot-cron"]
\ No newline at end of file
+["hubot-cron", "moment"]
```

Comment:
Unless this is a hubot script, you shouldn't have to add this here. The addition to `package.json` is all you need.